src/components/project/list.vue

```diff
@@ -224,3 +222,51 @@ export default {
   table-layout: fixed;
 }
 </style>
+
+<i18n lang="json5">
+{
+  "en": {
+    "heading": [
+      "Welcome to Central.",
+      "Let’s get some things done."
+    ],
+    "gettingStarted": {
+      // This is a title shown above a section of the page.
+      "title": "Getting Started",
+      "body": [
+        {
+          "full": "If you’re not sure where to begin, we have a getting started guide and user documentation available on the {docsWebsite}.",
+          "docsWebsite": "ODK Docs website"
+        },
+        {
+          "full": "In addition, you can always get help from others on the {forum}, where you can search for previous answers or ask one of your own.",
```

Comment:
This string doesn't really make sense. Maybe "search previous questions or ask one of your own"? Or "search through existing questions or ask a new one"? I think this would be nice to fix while we're here since it's on the landing page.